plugins/module_utils/foreman_helper.py

```diff
@@ -73,7 +73,9 @@
     #       Applying this will need to record a lot of tests that is out of scope for the moment.
     # organizations='title',
     scap_contents='title',
-    scc_products='friendly_name',
+    # NOTE: scc_product is searched by friendly_name before version v1.8.6.
+    # It is deprecated in version v1.8.6 and should not be used to search scc_product from version v1.8.6
+    # scc_products='friendly_name',
```

Comment:
I think a more accurate version of the comment would be something like:
```
# For SCC manager versions prior to v1.8.6 the 'name' field was not unique and 'friendly_name' was used instead.
# with version v1.8.6 this has been reversed (the 'name' field is now unique, but 'friendlly_name' is not).
```
We aren't really deprecating anything, we are simply dropping the `ENTITY_KEYS` redirect. As a result the newer version of the module won't work right for older versions of the SCC manager. I do not know how one could build this change in a backwards compatible way, and it is probably not worth the effort. Users will simply need to update their ansible modules and the SCC manager plugin in tandem...